src/Reflex/Dom/Builder/Class.hs

```diff
@@ -46,6 +46,7 @@ import Data.Text (Text)
 import Data.Traversable
 import Data.Type.Coercion
 import GHC.Exts (Constraint)
+import GHCJS.DOM.File (File)
```

Comment:
Yes; I think you should probably add a new type synonym to the DomBuilderSpace class.  Just think about what the `File` type ought to be for the Static builder - i'd guess either `()` or `Void`.